src/util/ConfigManager/ConfigManager.h

```diff
@@ -46,6 +46,32 @@ concept Validator =
     std::regular_invocable<Func, const ParameterTypes&...> &&
     std::same_as<std::invoke_result_t<Func, const ParameterTypes&...>, bool>;
 
+// Simple struct, that holds an error message. For use as the return type of
+// invocable object, that fullfill `ExceptionValidator`.
+struct ErrorMessage {
+  const std::string message_;
+
+  // Constructor.
+  ErrorMessage(std::string_view message) : message_{message} {}
```

Comment:
You need a string, so take a string by value and move it in.

Reply:
Done, see [e664eb4](https://github.com/ad-freiburg/qlever/pull/1086/commits/e664eb4b790056160b204e0028e957764f7be061).